### ISSUE TITLE
Unable to fetch factory when using a URL pointing to a GraphQL endpoint

### ISSUE THREAD

**Issue by ElizabethWoodhouse** [mizdra/graphql-codegen-typescript-fabbrica#87]
### Environment

@mizdra/graphql-codegen-typescript-fabbrica version: 
typescript version (optional): 
@graphql-codegen/cli version (optional): 
@graphql-codegen/typescript version (optional):

### Summary

codegen.ts looks like this - the typescript plugin works fine and generates types but the fabbrica plugin does not generate the appropriate information.

```ts
import type { CodegenConfig } from '@graphql-codegen/cli';

const config: CodegenConfig = {
  schema: 'graphQL-url-endpoint',
  debug: true,
  silent: false,
  verbose: true,
  generates: {
    '__generated__/types.ts': {
      plugins: ['typescript'],
      config: {
        enumsAsTypes: true, // required
        avoidOptionals: false, // required
      },
    },
    './__generated__/fabbrica.ts': {
      plugins: ['@mizdra/graphql-codegen-typescript-fabbrica'],
      config: {
        typesFile: './types', // required
        skipIsAbstractType: false,
      },
    },
  },
};

export default config;
```

### Step to reproduce

```ts
<!-- Paste your code here -->
```

### What did you expect to happen?

Expected fabbrica.ts to be generated based on our schema

### What actually happened?

Fabbrica file output this:
```ts
import {
  type Traits,
  type TypeFactoryDefineOptions,
  type TypeFactoryInterface,
  type FieldsResolver,
  defineTypeFactoryInternal,
} from '@mizdra/graphql-codegen-typescript-fabbrica/helper';
import type { Maybe,  } from './types';

export * from '@mizdra/graphql-codegen-typescript-fabbrica/helper';
```

Could not find anything in the error logs to indicate any issues with the set-up but any guidance would be appreciated!

### Participation

- [ ] I am willing to submit a pull request for this issue.

**Comment by moh682** [comment]
[Here](https://stackblitz.com/edit/playground-graphql-codegen-typescript-fabbrica-x2yy5z?file=codegen.ts) is an example from stackBlitz.

**Comment by mizdra** [comment]
Thanks for providing the reproduction code! I will look into the issue.

**Comment by mizdra** [comment]
@ElizabethWoodhouse @moh682 This is fixed by v0.6.1. Thanks for contributing!

https://github.com/mizdra/graphql-codegen-typescript-fabbrica/releases/tag/v0.6.1
